### ISSUE TITLE
[Question] Error on update YetiForce from 4.3.0 to 4.4.0

### ISSUE THREAD

**Issue by artyomeg** [YetiForceCompany/YetiForceCRM#11090]
### Description

```
[26-Jul-2019 23:19:58 Asia/Kuwait] PHP Fatal error:  Uncaught Error: Call to a member function getColumn() on null in C:\OSPanel\domains\crm4u.ru.os\cache\updates\updates\init.php:1010
Stack trace:
#0 C:\OSPanel\domains\crm4u.ru.os\cache\updates\updates\init.php(118): YetiForceUpdate->dav()
#1 C:\OSPanel\domains\crm4u.ru.os\vtlib\Vtiger\PackageImport.php(998): YetiForceUpdate->update()
#2 C:\OSPanel\domains\crm4u.ru.os\vtlib\Vtiger\PackageImport.php(552): vtlib\PackageImport->importUpdate()
#3 C:\OSPanel\domains\crm4u.ru.os\modules\Settings\ModuleManager\views\ModuleImport.php(139): vtlib\PackageImport->import('cache/vtlib/use...')
#4 C:\OSPanel\domains\crm4u.ru.os\include\runtime\Controller.php(90): Settings_ModuleManager_ModuleImport_View->importUserModuleStep3(Object(App\Request))
#5 C:\OSPanel\domains\crm4u.ru.os\modules\Settings\ModuleManager\views\ModuleImport.php(33): Vtiger_Controller->invokeExposedMethod('importUserModul...', Object(App\Request))
#6 C:\OSPanel\domains\crm4u.ru.os\include\main\WebUI.php(190): Settings_ModuleManager_ModuleImport_View->pr in C:\OSPanel\domains\crm4u.ru.os\cache\updates\updates\init.php on line 1010

```

**Comment by artyomeg** [comment]
I have an error when tried to update `4.3.0_to_4.4.0_v3.zip`.
`4.3.0_to_4.4.0_PreInstall.zip` was updated successfully.

How could i fix it and update my system to 4.4.0 and over?
Thank you for attention for my problem!

**Comment by rskrzypczak** [comment]
You probably have a faulty database structure already in 4.3.
I think you don't have the dav_calendars table.
If you don't have this table, then who knows what else you lack in the database.

What might be the case is that you ran the update package multiple times because of some errors that kept popping up.
Then running update from database backup, that you obviously have, might solve the issue. Check if the table I mentioned exists there.
